redfishMockupServer.py

```diff
@@ -515,20 +515,41 @@ def do_GET(self):
                 self.wfile.write(encoded_data)
 
             # if XML...
-            elif(os.path.isfile(fpath_xml) or os.path.isfile(fpath_direct)):
-                if os.path.isfile(fpath_xml):
-                    file_extension = 'xml'
-                    f = open(fpath_xml, "r")
-                elif os.path.isfile(fpath_direct):
-                    filename, file_extension = os.path.splitext(fpath_direct)
-                    file_extension = file_extension.strip('.')
-                    f = open(fpath_direct, "r")
+            elif os.path.isfile(fpath_xml):
+                f = open(fpath_xml, "r")
                 self.send_response(200)
-                self.send_header("Content-Type", "application/" + file_extension + ";odata.metadata=minimal;charset=utf-8")
+                self.send_header("Content-Type", "application/xml;odata.metadata=minimal;charset=utf-8")
```

Comment:
Can we remove the "odata.metadata=minimal" from here and below while we're at it? We've been moving away from these tags on content-type

Reply:
Done.